### DISCUSSION TITLE
Question about test set

### DISCUSSION THREAD

**Discussion by moonquakes66** [PaddlePaddle/PaddleSpatial#61] [BaiduKDDCup2022] [answered]
So does the test set'Day begin with 185 and end with 187?

**Comment by ramber1836** [comment]
No, it doesn't.
Please refer to our latest report for more details.

**Comment by DrownFish19** [comment, marked as answer]
No. I think you misunderstand the content of the report. In the report, the details are as follows:
> We use K data instances to evaluate the performance of the prediction model. For each data instances 𝑘, we randomly sample a stride time step Δ𝑡 𝑘 from the range [10 minutes, 100 minutes].

> For the evaluation in our submission system (starting from May 10), we use 195 random sampled stride time steps (i.e. {Δ𝑡 0 , Δ𝑡 1 , ..., Δ𝑡 194 } in Eqn. 3) over several months to evaluate the submitted models. This test time range and sampled stride time steps will be updated in future phases (June 20 and July 15).

That means that there are 195 files in the `data/sdwpf_baidukddcup2022_test_toy/test_x` and `data/sdwpf_baidukddcup2022_test_toy/test_y`, respectively. Specifically, the name of files are `0001in.csv, 0002in.csv,...,0195in.csv` and `0001out.csv, 0002out.csv,..., 0195out.csv`. Thus the range of test dataset is not begin with 185 and end with 187.

And, the test time range and sampled stride time steps will be updates on June 20 and July 15. We need to wait for Baidu to release more details.
